wpengine/faustjs
# Experimental App Router: Login server action

## Tasks


## Description

This PR adds the `onLogin` Next.js server action to log a user into Faust via the App Router.



## Related Issue(s):



## Testing

**If testing this PR with Safari, please ensure your Faust/next app is running on `https`**

1. Checkout `merl-1203-login-server-action`
2. `npm install`
3. Build the packages `npm run build`
4. Run the example project `npm run dev -w @faustwp/app-router-example`
5. Navigate to `http://localhost:3000/login`
6. Open your dev tools and notice **no** refresh token cookie 
7. Enter your WordPress username and password
8. Once you click the login button, upon successful login, the cookie will populate, and you will be redirected to `/gated-content` which shows the current logged in users posts

Video Demonstration:

https://github.com/wpengine/faustjs/assets/5946219/f98b7682-84c3-4674-83dd-96fd2c5f19f4



## Screenshots



## Documentation Changes



## Dependant PRs
